hack/local-setup.sh

```diff
@@ -224,6 +224,17 @@ initController() {
     ${KUSTOMIZE_BIN} --load-restrictor LoadRestrictionsNone build config/webhook-setup/proxy | kubectl apply -f -
 }
 
+# defaultGatewayParams adds a cluster to the config/samples/gatewayclass_params.json
+# file. This file can be used to create a ConfigMap referenced in the
+# parameterssRef section of the GatewayClass
+defaultGatewayParams() {
+  clusterName=${1}
+
+  echo "Adding ${clusterName} to gatewayclass params"
+
+  ${YQ_BIN} -i -P -o=json '.downstreamClasses["'$clusterName'"] = "istio"' config/samples/gatewayclass_params.json
```

Comment:
@sergioifg94 why are we keying it off the cluster name? The idea with the gatewayclass params is it is configuration to be applied to gateways of this class, so I would expect all gateways of this class would have the same downstream class